articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+compressing images before the upload is recommended. Photoshop,
+[compressor.io](https://compressor.io/) and [BulkResize](https://bulkresizephotos.com/en) are some of the tools used
+for image compression. Image compression plugins include [Imagify](https://imagify.io/wordpress/),
+[ShortPixel](https://wordpress.org/plugins/shortpixel-image-optimiser/),
+[Optimole](https://wordpress.org/plugins/optimole-wp/), and [WP
+Smush](https://wordpress.org/plugins/wp-smushit/). Keep your image under
+100 KB for the best performance.
+
+### Delete Unused Themes, Plugins, and Media Files
+
+This is a crucial website maintenance task. You should get rid of themes
+and plugins that are not in use. The plugins and themes take space in
+the server causing unnecessary strain. This involves deleting installed themes that are not in use and deleting deactivated plugins.
+
+### Use Well-optimized Themes
+
+Don't select some beautiful and impressive looking WordPress themes at
+the expense of performance. Pay special attention to theme optimization. Your
+WordPress theme should be elegant, lightweight, SEO friendly, fully
+responsive, and mobile optimized.
+
+### Optimize WordPress Database
+
+Over time, the WordPress database will accumulate unnecessary data,
+clutter. This will slow down your website. The clutter could be post
+revisions, data that was created by already uninstalled plugins, and
+WordPress sessions. To optimize the database, you can use plugins. It is
+advisable to back up your database before trying to clean. Some of the
+plugins available for database optimization are
+[WP-Optimize](https://wordpress.org/plugins/wp-optimize/) and
+[WP-Sweep](https://wordpress.org/plugins/wp-sweep/).
+
+![](wp-optimize-database-optimization.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version database
+optimization
+
+### Use Lazy Loading for Images and Videos
+
+Lazy loading is a web optimization technique whereby, visible content is
+loaded first while contents appearing down the fold download and
+rendering is delayed. Instead of loading all images and videos, Lazy
+loading downloads and render only what is visible to the user's screen.
+As the user scrolls down the page, more content is loaded. This
+significantly increases the loading speed of the content that is visible
+to the user.
+
+### Limit the Number of Post Revisions
+
+WordPress has a very good feature called post revisions. Whenever you
+update a page or a post, another version of the post is created and
+saved in the database. Overtime, post revisions can increase to a point
+whereby they slow your WordPress database. In the screenshot below, post
+revisions have accumulated to 93.
+
+![WordPress post revisions screenshot](wordpress-post-revisions-screenshot.jpeg)
+
+The solution for post revisions is to clean them from the database. You
+can use the WP-Optimize plugin to clean post revisions as shown in the
+screenshot below.
```

Comment:
Perhaps provide an explicit warning to the user that only four post revision will be saved so they will only be able to access the last four versions of their page or post.